.devcontainer/docker-start.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
Can you please add the generic shebang `#!/usr/bin/env bash` and the copyright headers to your scripts?

Reply:
Sure, updated!